src/Servers/Kestrel/Transport.Quic/src/QuicConnectionFactory.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Quic;
+using System.Net.Quic.Implementations;
+using System.Net.Security;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Connections;
+using Microsoft.AspNetCore.Server.Kestrel.Transport.Quic.Internal;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Quic
```

Comment:
I think we should either remove ".Client" from Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.Client.SocketConnectionFactory or add it here. Fortunately SocketConnectionFactory hasn't shipped yet, so we have a choice.